docs/rest_api.md

```diff
@@ -416,6 +416,12 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
```

Comment:
If we had more than one or two OBM drivers (like we do for network drivers), I'd agree with you, though for now I'm comfortable inlining these until there are more.

@zenhack - what do you think?